From bc7b8b21549752274b162c64dbce3c81d20f2cab Mon Sep 17 00:00:00 2001
From: Stewart Smith <stewart@linux.vnet.ibm.com>
Date: Thu, 25 Aug 2016 20:22:30 +1000
Subject: [PATCH 03/10] Default to std=gnu++03

Seeing as no C++ dialect was previously selected, GCC < 6 defaulted
to GNU++98 as the standard C++ mode. However... it seems that C++03
is a better match for HostBoot code (confirmed by Patrick Williams
in https://github.com/open-power/hostboot/pull/62#discussion_r76830612 )

Change-Id: I6661b5b61319c85c2a90926beb6e2662e96dcf06
Signed-off-by: Stewart Smith <stewart@linux.vnet.ibm.com>
---
 src/build/mkrules/cflags.env.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/build/mkrules/cflags.env.mk b/src/build/mkrules/cflags.env.mk
index c805999..0bae0ec 100644
--- a/src/build/mkrules/cflags.env.mk
+++ b/src/build/mkrules/cflags.env.mk
@@ -41,7 +41,7 @@ CFLAGS += $(COMMONFLAGS) -mcpu=power7 -nostdinc -g -mno-vsx -mno-altivec\
 	  -ffunction-sections -fdata-sections -ffreestanding -mbig-endian
 ASMFLAGS += $(COMMONFLAGS) -mcpu=power7 -mbig-endian
 CXXFLAGS += $(CFLAGS) -nostdinc++ -fno-rtti -fno-exceptions -Wall \
-	    -fuse-cxa-atexit
+	    -fuse-cxa-atexit -std=gnu++03
 LDFLAGS += --nostdlib --sort-common -EB $(COMMONFLAGS)
 
 INCFLAGS = $(addprefix -I, $(INCDIR) )
-- 
2.7.4

